src/models/transactions/common.ts

```diff
@@ -48,13 +48,17 @@ const isSigner = (signer: Signer): boolean => {
         && typeof signer.SigningPubKey === 'string'
 }
 
-export interface CommonFields {
+export interface GlobalFlags {
+    tfFullyCanonicalSig: boolean,
```

Comment:
With [`RequireFullyCanonicalSig`](https://xrpl.org/known-amendments.html#requirefullycanonicalsig), this flag has no real effect. It's ok to leave it, but I think we might be able to save a few lines of code by leaving it out. When we `prepare` or `autofill` a transaction, I think it's still a good idea to set the flag, just in case.

Reply:
https://xrpl.org/transaction-common-fields.html#global-flags